modules/runtime/examples/docker/cluster/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+# Example on how to use Postgres 15 with Bonita in a cluster mode
+services:
+  bonita-db:
+    image: bonitasoft/bonita-postgres:15.4
```

Comment:
15.4 does not exists

Reply:
https://github.com/Bonitasoft-Community/bonita-database-docker/pull/12